sympy/core/relational.py

```diff
@@ -1075,8 +1087,42 @@ def _n2(a, b):
             return dif
 
 
+@dispatch(Number, Pow)
+def _eval_is_ge(lhs, rhs): # noqa:F811
+    if lhs:
+        base, exp = rhs.as_base_exp()
+        if exp.is_nonpositive and is_ge(base, 1/lhs):
+            return True
+
+
+@dispatch(Pow, Number)
+def _eval_is_ge(lhs, rhs): # noqa:F811
+    if rhs:
+        base, exp = lhs.as_base_exp()
+        if exp.is_nonpositive and is_le(base, 1/rhs):
+            return True
+
+
+@dispatch(Number, Mul)
+def _eval_is_ge(lhs, rhs): # noqa:F811
+    from sympy.simplify.radsimp import fraction
+    num, den = fraction(rhs)
+    if num.is_Number and den != 1:
+        if is_le(den, lhs/num):
+            return True
+
+
+@dispatch(Mul, Number)
+def _eval_is_ge(lhs, rhs): # noqa:F811
+    from sympy.simplify.radsimp import fraction
+    num, den = fraction(lhs)
+    if num.is_Number and den != 1:
+        if is_ge(den, rhs/num):
+            return True
```

Comment:
```suggestion
```suggestion
    if num and num.is_Number and den != 1:
        a, b = den, rhs/num
        if (num < 0):
            a, b = b, a
        if is_ge(a, b):
            return True
``````